File: programmers/42576.py
def solution(participant, completion):
    dic={}
    sum = 0

    for i in participant:
        dic[hash(i)] = i
        sum += hash(i)
    
    for i in completion:
        sum -= hash(i)
    
    answer = dic[sum]

    return answer

# print(solution(participant=["mislav", "stanko", "mislav", "ana"],completion=["stanko", "ana", "mislav"]))

File: programmers/42576.js
function solution(participant, completion) {
    const all = {};

    for (let p of participant){
        all[p] = all[p] ? all[p]+1 : 1;
    }
    for (let c of completion){
        all[c]--;
    }
    for (let key in all){
        if(all[key]===1){
            return key;
        }
    }
}

console.log(solution(["leo", "kiki", "eden"],["eden", "kiki"]))